src/main/java/com/shrek/task00001/Main.java

```diff
@@ -9,10 +8,66 @@
  */
 public class Main {
     public String getWinner(String[] runners, int[] times) {
-        throw new NotImplementedException();
+
+        String winnerName;
+        if (times.length == 0) {
+            return null;
+        }
+        int min1 = times[0];
+
+        for (int i = 0; i <= times.length - 1; i++) {
+            if (times[i] < min1) {
+                min1 = times[i];
+            }
+        }
+        int winnerPos = 0;
+        for (int pos = 0; pos <= times.length - 1; pos++) {
+            if (times[pos] == min1) {
+                winnerPos = pos;
+            }
+        }
+        winnerName = runners[winnerPos];
+        return winnerName;
     }
 
     public String getSecond(String[] runners, int[] times) {
```

Comment:
1. Please use more clear names for variables. min1, min2, min3... what are they? 
2. It also may be done with one cycle :) I can't understand why you find minimum 2 times.